### PR TITLE
Add frontmatter fields to the export

### PR DESCRIPTION
**Pull request by devnook**

### CODE REVIEW

File: src/site/_includes/components/Export.js [hunk 1]
```diff
@@ -64,6 +64,18 @@ async function Export() {
...
   }
 
+  const authors = JSON.stringify(page?.template?.frontMatter?.data.authors);
+
+  const frontMatter = `project_path: /_project.yaml
+book_path: /_book.yaml
+
+{% import "/_macros.html" as macros %}
+
+# ${page?.template?.frontMatter?.data.title}
```
Comment (matthiasrohmer) [inline src/site/_includes/components/Export.js:74]: Does this get the computed title? Haven't checked if there are cases where we use one.

File: src/site/_includes/components/Export.js [hunk 2]
```diff
@@ -64,6 +64,18 @@ async function Export() {
     return '';
   }
 
+  const authors = JSON.stringify(page?.template?.frontMatter?.data.authors);
+
+  const frontMatter = `project_path: /_project.yaml
+book_path: /_book.yaml
+
```
Comment (matthiasrohmer) [inline src/site/_includes/components/Export.js:71]: There is also `author_name`, used in dynamic content cards.

Comment (devnook) [reply src/site/_includes/components/Export.js:71]: true, we can add it

File: src/site/_includes/components/Export.js (hunk 1 above)
Comment (devnook) [reply src/site/_includes/components/Export.js:74]: It does not. We don't use computed titles for posts though.

**Pull request merged**